static/js/board_carousel.js

```diff
@@ -149,33 +152,54 @@ function deleteElement(bobid){
   if ($('#slideshow').hasClass('initialized')) {
     $('#slideshow').removeClass('initialized');
   }
-
   // reinit the carousel
   $('#slideshow').carousel({
-    fullWidth: true
+    fullWidth: true,
+    onCycleTo: function(activeItem){
+      updateVoteLabel(activeItem);
+      loadVideo(activeItem);
+    }
   });
   // force move to next slide
   carouselControl("right");
 }
 
 /**
-  * Updates the label with the votes label for currently active bob
+  * @param {object} activeItem - Vanilla JS Object of activeItem
+  * Updates the label with the votes label for currently activeItem
 */
-function updateVoteLabel(){
+function updateVoteLabel(activeItem){
   var $labelToUpdate = $("#votes");
-  var activeBobID = $("#slideshow").find(".active").attr("id");
-  var votes =  $.get('/api/bobs/' + activeBobID + "/votes", function(res){
+  var $activeBobID = ($(activeItem).attr("id"));
```

Comment:
the parenthesis aren't necessary, but $ is necessary because the argument is a Vanilla JS object returned from materialize carousel method